_monitoring-your-cluster/pa/reference.md

```diff
@@ -455,225 +531,498 @@ This list is extensive. We recommend using Ctrl/Cmd + F to find what you're look
+     </td>
+     <td>The maximum heap usage for tasks currently running on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_heap_usage_rollingAvg
+     </td>
+     <td> The rolling average heap usage for n most recent tasks on the shard task. The default value for n is 100.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_cpu_usage_cancellationCount
+     </td>
+     <td>he number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_cpu_usage_currentMax
+     </td>
+     <td>The maximum CPU time for all tasks currently running on the node on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_cpu_usage_currentAvg
+     </td>
+     <td>The average CPU time for all tasks currently running on the node on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_elaspedtime_usage_cancellationCount
+     </td>
+     <td>The number of tasks marked for cancellation because of excessive elapsed time since the node last restarted on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_elaspedtime_usage_currentMax
+     </td>
+     <td>The maximum elapsed time for all tasks currently running on the node on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_shard_stats_resource_elaspedtime_usage_currentAvg
+     </td>
+     <td>The average elapsed time for all tasks currently running on the node on the shard task.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_task_stats_cancellationCount
+     </td>
+     <td>The number of tasks marked for cancellation on the search task level.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_task_stats_limitReachedCount
+     </td>
+     <td>The number of times when the number of tasks eligible for cancellation exceeded the set cancellation threshold on the search task level.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_task_stats_resource_heap_usage_cancellationCount
+     </td>
+     <td>The number of tasks marked for cancellation because of excessive heap usage since the node last restarted on the search task level.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_task_stats_resource_heap_usage_currentMax
+     </td>
+     <td>The maximum heap usage for tasks currently running on the search task level.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_task_stats_resource_heap_usage_rollingAvg
+     </td>
+     <td> The rolling average heap usage for n most recent tasks on the search task level. The default value for n is 10.
+     </td>
+   </tr>
+   <tr>
+     <td>searchbp_task_stats_resource_cpu_usage_cancellationCount
+     </td>
+     <td>he number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the search task level.
```

Comment:
```suggestion
     <td>The number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the search task level.
```